pacemaker.spec.in

```diff
@@ -630,6 +653,7 @@ exit 0
 
 %config(noreplace) %{_sysconfdir}/sysconfig/pacemaker
 %if %{defined _unitdir}
+%ghost %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
Oh, right. So we can't ghost the file unless the ghost removal (for the old version in an upgrade) is guaranteed not to happen between when the new version touches the file and when it restarts pacemaker_remote.